tests/system/test_integration_acm.py

```diff
@@ -254,3 +274,91 @@ def test_log_kill_switch_active(self):
             "error": "forbidden request: endpoint is disabled",
             "response_code": 403,
         }, config_request_logs[0])
+
+
+class RumAgentConfigurationIntegrationTest(AgentConfigurationTest):
+    config_overrides = {
+        "enable_rum": "true",
+    }
+
+    def test_rum(self):
+        service_name = "rum-service"
+        create_config_rsp = self.create_service_config({"transaction_sample_rate": 0.2}, service_name, agent="rum-js")
+        create_config_rsp.raise_for_status()
+        assert create_config_rsp.status_code == 200, create_config_rsp.status_code
+        create_config_result = create_config_rsp.json()
+        assert create_config_result["result"] == "created"
+
+        r1 = requests.get(self.rum_agent_config_url,
+                          params={"service.name": service_name},
+                          headers={"Content-Type": "application/json"})
+
+        assert r1.status_code == 200
+        assert r1.json() == {'transaction_sample_rate': '0.2'}
+        etag = r1.headers["Etag"]
+
+        r2 = requests.get(self.rum_agent_config_url,
+                          params={"service.name": service_name, "ifnonematch": etag.replace('"', '')},
```

Comment:
why is it necessary to remove surrounding `"` of etag? It should be possible to just use the etag that is returned by the server.

Reply:
because the rum agent will actually send it without double quotes

Reply:
But shouldn't the server be able to process with or without quotes? And why is the RUM agent sending without quotes (I probably missed that discussion somewhere).

Reply:
I approved the PR, as I haven't followed discussions about this. Please ensure then RUM agent properly removes the quotes from the response they get if you don't change it in the server.